example/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "demo",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "react": "^16.6.3",
+        "react-dom": "^16.6.3",
+        "react-scripts": "2.1.1"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject",
+        "link:nice2": "cd .. && npm link && cd ./example && npm link nice2"
+    },
+    "browserslist": [
```

Comment:
Consider having this in sync with our browser support guidelines: https://www.textkernel.com/browser-support/